src/js/utils/DOM.js

```diff
@@ -157,7 +157,7 @@ export const makeNodeFocusable = (node) => {
   }
 };
 
-const autoFocusingTags = /(a|area|input|select|textarea|button|iframe)$/;
+const autoFocusingTags = /^(a|area|input|select|textarea|button|iframe)$/;
```

Comment:
can we add a comment? just for future reference 
`maybe something like to avoid unintentionally matching <data and <meta tags we need the ^`

Reply:
Added, though it's just essential regex knowledge